falconx/client.py

```diff
@@ -360,6 +360,48 @@ def get_total_balances(self):
         response = self.session.get(self.url + 'balances/total')
         return self._process_response(response)
 
+    def get_derivatives(self, **kwargs):
+        """Get all derivative trade data with current mark-to-market data.
```

Comment:
For consistency sake, I would follow the comment blocks of the other functions.
```suggestion
        """
        Get all derivative trade data with current mark-to-market data.
```

Reply:
I thought about this too and erroneously went with the Google code style we follow in the other repo - I'll change it